src/main/java/frc/robot/Constants.java

```diff
@@ -100,6 +101,22 @@ public static class SwerveSystemDeviceConstants {
         }
     }
 
+    public static class VisionConstants {
+        /**
+         * Angle of camera pointing upwards.
+         */
+        public static final double limelightMountAngleRadiansY = 0;
```

Comment:
It would make more sense to uses degrees.

Reply:
Yes, I'll add this to each subsystem. It can be useful, however, to display radians as well when making offsets.